src/common/components/episode-list/episode-list.vue

```diff
@@ -130,40 +130,34 @@ export default class EpisodeList extends Vue {
   }
 
   @Watch('open')
-  @Watch('progress')
   public _scrollToNextEpisode(iteration: number = 0): void {
-    if (!this.$refs.episodes?.[0] && iteration < 50) {
+    if (!this.$refs.episodes && iteration < 50) {
       setTimeout(() => this._scrollToNextEpisode(iteration + 1), 50)
 
       return
     }
 
     if (
-      !this.scrollToNextEpisode ||
-      !this.open ||
```

Comment:
These should probably still be here, no?

Reply:
These two were always showing up false, I didn’t see them being set I’ll check in the parent component though.

Reply:
Yea, I don't see these properties being set, would you like me to implement them? Unless there's some hidden Vue magic being done behind the scenes.

Reply:
As it has the `@Prop()` decorator (L68) it gets the property set by parents when they use it.

The component is [used in two places](https://i.imgur.com/Xpa6MZH.png), and only one of them needs to pass the `open` property: the [QueueItem](https://github.com/BeeeQueue/yuna/blob/master/src/views/queue/components/queue-item.vue#L27-L35).

Reply:
Ah okay, I see so the open only applies when viewing the 'QueueItem' and the main episode page isn't one so the prop becomes false since it is closed. Is it cool to keep that out and remove `@Watch('open')` above `_scrollToNextEpisode`.

Reply:
I'm confused what you mean. We need `@Watch(open)` so it scrolls to the correct episode when you open a QueueItem.

Reply:
Oh yes you're right, not sure what I was thinking there. Sorry about that!

Reply:
Then if these are added back I think this PR is good to go

Reply:
Actually I'm not sure I like the `scrollToItem()` usage, I'll see if I can improve that and I'll fix this comment at the same time

Reply:
There was also a `scrollToPosition()` that took a scroll amount.

Reply:
It's bad practice to use internals of other components, they are internals for a reason. I realised I'd made a perfect `scrollToEpisode` function in another component, so I simply moved it up and used it instead.

Reply:
Oh okay, I see, good to know for the future thank you!